README.md

```diff
@@ -1,5 +1,119 @@
-# THIS PROJECT NEEDS A MAINTAINER.
+# angular-sortablejs
 
----
+This package is an Angular 2 binding for awesome [Sortable.js](https://github.com/RubaXa/Sortable) library which supports both Webpack and SystemJS.
 
-# HELP WANTED ;]
+## Installation
+
+    npm install --save angular-sortablejs
+
+Note: you **do not** need to install Sortable.js! It will be installed automatically.
+
+## Usage
+
+### Example
+
+First, include `SORTABLEJS_DIRECTIVES` into your component:
+
+    import { Component } from '@angular/core';
+    import { SORTABLEJS_DIRECTIVES } from 'angular-sortablejs';
+
+    @Component({
+        selector: 'my-app',
+        template: `
+          <h2>Drag / drop the item</h2>
+          <div [sortablejs]="items">
```

Comment:
I'm not really sure about that. Angular2 is quite different, at the moment I don't know the way.